result/ospfv2lsa.yaml

```diff
@@ -0,0 +1,339 @@
+          type: array
+          items:
+            $ref: '#/components/schemas/Ospfv2.RouterLsa'
+          x-field-uid: 3
+        network_lsas:
+          description: >-
+            One or more OSPFv2 Network-LSA - Type 2.
+          type: array
+          items:
+            $ref: './ospfv2lsa.yaml#/components/schemas/Ospfv2.NetworkLsa'
+          x-field-uid: 4
+        network_summary_lsas:
+          description: >-
+            One or more OSPFv2 Network summary LSA - Type 3.
+          type: array
+          items:
+            $ref: '#/components/schemas/Ospfv2.NetworkSummaryLsa'
+          x-field-uid: 5
+        summary_as_lsas:
+          description: >-
+            One or more OSPFv2 Autonomous System Boundary Router (ASBR) summary LSA - Type 4.
+          type: array
+          items:
+            $ref: './ospfv2lsa.yaml#/components/schemas/Ospfv2.SummaryAsLsa'
+          x-field-uid: 6
+        external_as_lsas:
+          description: >-
+            OSPFv2 AS-External-LSA - Type 5.
+          type: array
+          items:
+            $ref: './ospfv2lsa.yaml#/components/schemas/Ospfv2.ExternalAsLsa'
+          x-field-uid: 7
+        nssa_lsas:
+          description: >-
+            One or more OSPFv2 NSSA-LSA - Type 7.
+          type: array
+          items:
+            $ref: './ospfv2lsa.yaml#/components/schemas/Ospfv2.NssaLsa'
+          x-field-uid: 8
+        opaque_lsas:
+          description: >-
+            One or more OSPFv2 Link-Scope Opaque-LSA - Type 9.
+          type: array
+          items:
+            $ref: './ospfv2lsa.yaml#/components/schemas/Ospfv2.OpaqueLsa'
+          x-field-uid: 9
+    Ospfv2.RouterLsa:
+      description: >-
+        Contents of the router LSA
+      type: object
+      properties:
+        common_attrs:
+          description: >-
+            Contents of the router LSA.
+          $ref: './ospfv2lsa.yaml#/components/schemas/Ospfv2.CommonAttrs'
+          x-field-uid: 1
+        links:
+          description: >-
+            Links that are described within the LSA.
+          type: array
+          items:
+            $ref: '#/components/schemas/Ospfv2.Link'
+          x-field-uid: 2
+    Ospfv2.NetworkLsa:
+      description: >-
+        Contents of the Network LSA
+      type: object
+      properties:
+        common_attrs:
+          description: >-
+            Contents of the router LSA.
+          $ref: './ospfv2lsa.yaml#/components/schemas/Ospfv2.CommonAttrs'
+          x-field-uid: 1
+        network_mask:
+          description: >-
+            The IPv4 address mask for the network.
+          type: string
+          format: ipv4
+          x-field-uid: 2
+        neighbor_router_id:
+          description: >-
+            Neighbor's Router ID for the link.
+          type: string
+          format: ipv4
+          x-field-uid: 3
+    Ospfv2.NetworkSummaryLsa:
+      description: >-
+        Contents of the Network Summary LSA - Type 3.
```

Comment:
"The value of the IPv4 prefix that was received is present in header.lsa_id. ( similar for External AS and NSSA LSAs) " to clarify that the main attribute of interest ( the route ) is hidden in the common  common_attrs/header field.

Reply:
Done.